.github/workflows/docs.yml

```diff
@@ -20,9 +20,26 @@ on:
 jobs:
   doxygen:
     runs-on: ubuntu-16.04
+    env:
+      CXX: g++
+    container:
+      image: aswf/ci-openvdb:2019
     steps:
     - uses: actions/checkout@v2
-    - name: install
-      run: ./ci/install_doxygen.sh
+    - name: install_doxygen
+      run: ./ci/install_doxygen.sh 1_8_11
+    - name: install_epydoc
+      run: pip install epydoc
+    - name: install_latex
+      run: yum -y install texlive-latex-bin texlive-dvips
     - name: build
-      run: ./ci/build.sh Release None OFF None "doc" -DCMAKE_INSTALL_PREFIX=$HOME
+      run: ./ci/build.sh Release None OFF None "core,python,doc" -DOPENVDB_PYTHON_WRAP_ALL_GRID_TYPES=ON -DDISABLE_DEPENDENCY_VERSION_CHECKS=ON
+    - name: mkdir
+      run: mkdir -p $HOME/docs
+    - name: install
+      run: cp -r /usr/local/doc/html/* $HOME/docs
```

Comment:
is this dir guaranteed to always only contain the vdb installed docs?

Reply:
Yes - for now this just gets thrown away when the job finishes, but the next step is to try and publish this somewhere so we don't have to ever generate documentation offline.

Reply:
sure, I was just concerned that another dep or pkg might install stuff here that we'd end up copying

Reply:
It doesn't seem to be the case right now, but I can rename this to make this location more likely to be unique if you're concerned.

Reply:
I think it's actually `/usr/share/doc` I'm thinking of (maybe we should be installing there!) but a unique location would be good

Reply:
Ah, I thought you were talking about the target ($HOME/docs). I see what you mean, other packages could feasibly have installed something inside an html directory here. I'll look to change the install prefix.

Reply:
For future consideration:  https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html has been updated to include `CMAKE_INSTALL_DOCDIR` which evals to `share/doc/OpenVDB` on my system

Reply:
ok, thanks - we could use this in the future, for now I just changed the install location